static/js/context-menu.js

```diff
@@ -184,10 +188,12 @@ $("#file").on("click", "code", function(event) {
                     icon: "search"});
   }
 
-  var symbols = elt.closest("[data-symbols]").attr("data-symbols");
+  var token = elt.closest("[data-symbols]");
+  var symbols = token.attr("data-symbols");
+  var visibleToken = token[0].textContent;
```

Comment:
Hmm, doesn't this throw when you're not hovering over something with symbols? `$().attr("foo")` returns undefined, but `$()[0].foo` throws. You should probably move this line under the `if`.

Reply:
Yeah, I just assumed that if we're in this part of the code we'd find something. But your suggestion is better, I moved it inside the if.